agent/app/agent_capability.go

```diff
@@ -498,6 +503,24 @@ func (agent *ecsAgent) appendServiceConnectCapabilities(capabilities []*ecs.Attr
 	return capabilities
 }
 
+func (agent *ecsAgent) appendGuardDutyCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
```

Comment:
Q. Should we define this in [agent_capability_unix.go](https://github.com/aws/amazon-ecs-agent/blob/master/agent/app/agent_capability_unix.go) since GuardDuty will not be supported on windows and unspecified platforms?